plugins/runtime_id_plugin/runtime_id_plugin.c

```diff
@@ -0,0 +1,36 @@
+#include "runtime_id_plugin.h"
+
+#include <sodium.h>
```

Comment:
From our conversation from earlier, I think we need to vendor this in for the short term.